backend/api/swagger/pipeline.swagger.json

```diff
@@ -593,7 +613,8 @@
       "type": "object",
       "properties": {
         "pipeline_url": {
-          "type": "string"
+          "type": "string",
+          "description": "Online storage URL of the file containing the pipeline or pipeline version definition."
```

Comment:
Suggested revision:

URL of the pipeline definition or the pipeline version definition.

Reply:
done. Changed it in the proto.